include/flecs.h

```diff
@@ -2362,7 +2367,43 @@ void _ecs_assert(
 
 /** Calculate offset from address */
 #define ECS_OFFSET(o, offset) (void*)(((uintptr_t)(o)) + ((uintptr_t)(offset)))
+#define is_empty(...) ( sizeof( (char[]){#__VA_ARGS__} ) == 1 )
```

Comment:
Is this portable?

Reply:
I remove it since I'm not using macros anymore.